### PR TITLE
add route "Generate Court Report" to sidebar

### LINKED ISSUE #923: Add route to "Generate Court Report" to sidebar
Part of epic #880 // dependent on #922 

AS a volunteer, 
I want to click "Generate Court Report" in the left sidebar,
so that I can select the `casa_case` I am generating a court report for, and then generate and download said report.

**Acceptance Criteria**
- [ ] Text: "Generate Court Report" displays in left sidebar.
- [ ] "Generate Court Report" is clickable.
- [ ] When clicked, "Generate Court Report" form displays in container (the same way all other forms appear on platform).

### PR DESCRIPTION
**Pull request by erik-trantt**
### What github issue is this PR for, if any?
Resolves #923 

### What changed, and why?
- Added a route to "Generate Court Report": `<domain_name>/case_court_reports/`, and placed it to the sidebar.
- When visit "Generate Court Report" link, users can see a form with a select element showing their actively assigned cases.
- The form also has 2 buttons: "Generate Report" & "Download Court Report" as specified in #922 acceptance criteria.
  - The buttons do nothing for now. The functionalities are to be developed in #922. 

### How will this affect user permissions?
N/A
Currently, all users can see visit the link and see the form.

### How is this tested? (please write tests!) 💖💪
- 1 test to check if link 'Generate Court Report' on sidebar is generated
  - `spec/views/layouts/sidebar.html.erb_spec.rb`
- Tests to check if the page loads and the form elements (case selection & buttons) are present
  - `app/views/case_court_reports/index.html.erb`

### Screenshots please :)
image
image
image

### CODE REVIEW

File: app/views/case_court_reports/index.html.erb [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<h1>Report Categories</h1>
+
+<div class="card card-container">
+  <div class="card-body">
+    <h5 class="card-title"><strong>Generate Court Report</strong></h5>
+    <p class="card-text">
+      The Court Report is pre-filled with information of your case. You can select among currently active cases assigned to you. The document is in Microsoft Word format (.dox).
```
Comment (compwron) [inline app/views/case_court_reports/index.html.erb:7]: Is `.dox` correct? I would expect `.doc` or `.docx` but I might be out of touch with windows

"information of your case" sounds funny - maybe "information *for* your case" instead?

**Comment by compwron** [comment]
Looking good! I see that this is still in "draft" state so I won't review it yet :)

File: app/views/case_court_reports/index.html.erb (hunk 1 above)
Comment (erik-trantt) [reply app/views/case_court_reports/index.html.erb:7]: @compwron You're right. It's a typo. The correct type is `.docx`.
I'll also change the wordings to "information for your case".

**Comment by erik-trantt** [comment]
The commit was in CI build queue last night, so I waited till it finished to "open" the PR. Is it okay to open while CI build hasnt finished yet?

> Looking good! I see that this is still in "draft" state so I won't review it yet :)

**Comment by compwron** [comment]
> The commit was in CI build queue last night, so I waited till it finished to "open" the PR. Is it okay to open while CI build hasnt finished yet?
> 
> > Looking good! I see that this is still in "draft" state so I won't review it yet :)

@erik-trantt it is ok to open the PR without a green build on this repo. :)

**Review by compwron — APPROVED** [review]
:)

**Comment by erik-trantt** [comment]
@compwron Thanks for the clarification 👍

**Pull request merged**